frontend/src/metabase/admin/people/components/NudgeToPro.styled.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+import ExternalLink from "metabase/core/components/ExternalLink";
+
+export const NudgeCard = styled.div`
+  background-color: ${color("bg-light")};
+  border-radius: 6px;
+  padding: 20px 24px;
+  margin-top: 32px;
```

Comment:
For vertical margins it's usually better to use rem units. They are more accessible for this use case since they scale with the default font size the user has set in their browser. Here's a neat [demo](https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/#vertical-margins-8) of this.